### PR TITLE
ARC-164 humble migration

### PR DESCRIPTION
**Pull request by MishkaMN**
# PR Details
## Description
Humble migration

## Related GitHub Issue
NA

## Related Jira Key
ARC-164

## Motivation and Context
Humble migration

## How Has This Been Tested?
Currently just testing if it builds.
TODO: Vehicle testing (from JIRA):
```
Consider running this standalone docker container on a vehicle and [1] verifying message publication through ros 2 topic echo and [2] verifying point cloud data is displayed in rviz. By this point, rviz should be included in the carma-base-humble image, so it should be possible to launch the application.

NDT Matching will be tested/verified at a later time as part of regression testing once carma-platform is fully ported to ROS 2 Humble
```

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [X] I have added any new packages to the sonar-scanner.properties file
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: docker/install.sh [hunk 1]
```diff
@@ -14,22 +14,31 @@
...
+    sudo sed -i 's/\beigen\b/Eigen3/g' /opt/ros/humble/share/velodyne_pointcloud/cmake/ament_cmake_export_dependencies-extras.cmake
+    sudo sed -i 's/\bpcl\b/PCL/g' /opt/ros/humble/share/velodyne_pointcloud/cmake/ament_cmake_export_dependencies-extras.cmake
+    
+    # Install the driver and pointcloud conversion package
+    sudo apt install ros-humble-velodyne-driver -y
 
-# Install the driver and pointcloud conversion package
-sudo apt install ros-foxy-velodyne-driver -y
+    source /opt/ros/humble/setup.bash
+    source /opt/autoware.ai/ros/install/setup.bash
```
Comment (john-chrosniak) [inline docker/install.sh:40]: We shouldn't need to source here since we already do one lines 23-24

Comment (MishkaMN) [reply docker/install.sh:40]: Don't we need to source after installing the package? Or does it source automatically

Comment (MishkaMN) [reply docker/install.sh:40]: Or I guess we should only source here and not above

Comment (john-chrosniak) [reply docker/install.sh:40]: Yeah good point, should just need to remove the above source then

**Review by john-chrosniak — APPROVED** [review]

**Pull request merged**